test/verifier.integration.spec.ts

```diff
@@ -15,9 +16,12 @@ describe('Verifier Integration Spec', () => {
   const providerStatesSetupUrl = `${providerBaseUrl}/provider-state`;
   const pactBrokerBaseUrl = `http://localhost:${PORT}`;
   const monkeypatchFile: string = path.resolve(__dirname, 'monkeypatch.rb');
```

Comment:
Is this test here to show the logger output? Or should it go?

Reply:
No, it's not here for that. I left it mostly because monkeypatch shouldn't make anything explode (it just warns now). But I think it's pretty low value. I'd be happy to have it removed.

Reply:
(especially as `monkeypatch.rb` is not where this test says it is)

Reply:
I removed monkeypatch.rb, but left the test. Not completely convinced it's necessary, though